_templates/contact.html

```diff
@@ -5,3 +5,6 @@
     <li><a href="https://github.com/esphome/esphome" target="_blank">Source Code</a></li>
 </ul>
 
+<a href="https://www.netlify.com">
```

Comment:
```suggestion
<a href="https://www.netlify.com" style="border-bottom: none">
```

Reply:
Thanks, I just took the example as it was =)